Week3/homework/4-step3.js

```diff
@@ -1,2 +1,5 @@
-'use strict';
-// paste your freeCodeCamp solutions in here
+// 3.4 Some practice with objects:
+
+//     https://www.freecodecamp.com/challenges/construct-javascript-objects-with-functions
+
+// Link doesn't work
```

Comment:
Fair enough.  I actually went to the site and searched for the `javascript`, `objects`, `function` and found a new page, but without a challenge.

Basically, you just needed to show that you can create an object using a function.